src/ompl/base/src/PlannerTerminationCondition.cpp

```diff
@@ -222,7 +222,12 @@ ompl::base::PlannerTerminationCondition ompl::base::plannerAndTerminationConditi
 
 ompl::base::PlannerTerminationCondition ompl::base::timedPlannerTerminationCondition(double duration)
 {
-    return PlannerTerminationCondition(boost::bind(&timePassed, time::now() + time::seconds(duration)));
+    return timedPlannerTerminationCondition(time::seconds(duration));
+}
+
+ompl::base::PlannerTerminationCondition ompl::base::timedPlannerTerminationCondition(time::duration duration)
+{
+    return PlannerTerminationCondition(boost::bind(&timePassed, time::now() + duration));
```

Comment:
Well here's a case where I thought they fit nicely on top of each other.
The existing `PTC(double)` call converts the value into a `duration` internally, so all I was doing was breaking out that intermediate step. Then if one was using `duration`, they didn't have to convert to a `double` just to have the `PTC` convert back to a `duration`... That seemed likely to lose timing resolution.